### DISCUSSION TITLE
Add `strategy` option to anchorable elements.

### DISCUSSION THREAD

**Discussion by BasilLangevin** [livewire/flux#922] [Feature Requests]
This would let users choose between `absolute` (default) and `fixed` strategies for positioning anchored popovers.

For example:
`<flux:dropdown strategy="fixed">`

This would solve anchoring issues in fixed containers (such as sidebars):

Without the fixed strategy, if the main window is scrolled while an anchored popover is open in a fixed container, the popover scrolls with the page rather than sticking to the anchor.

**Comment by jeffchown** [comment]
@BasilLangevin Have you seen this issue?: https://github.com/livewire/flux/issues/110 and updated Flux to the latest release v1.1.2?
